DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/AddDelegateToGroup/AddDelegateToGroupViewModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.AddDelegateToGroup
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Models.CustomPrompts;
+    using DigitalLearningSolutions.Data.Models.User;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.AllDelegates;
+
+    public class AddDelegateToGroupViewModel : AllDelegatesViewModel

```

Comment:
Should we be inheriting from `AllDelegatesViewModel` ?
I don't believe the filtering/sorting AC's are eqivalent.

Reply:
This has now been refactored.